airbyte-integrations/connectors/source-oracle/src/main/java/io/airbyte/integrations/source/oracle/OracleSource.java

```diff
@@ -31,18 +33,33 @@
 
   private List<String> schemas;
 
+  private static final String KEY_STORE_FILE_PATH = "clientkeystore.jks";
+  private static final String KEY_STORE_PASS = RandomStringUtils.randomAlphanumeric(8);
+
+  enum Protocol{
+    TCP,TCPS
+  }
+
   public OracleSource() {
     super(DRIVER_CLASS, new OracleJdbcStreamingQueryConfiguration());
   }
 
   @Override
   public JsonNode toDatabaseConfig(JsonNode config) {
-    final ImmutableMap.Builder<Object, Object> configBuilder = ImmutableMap.builder()
-        .put("username", config.get("username").asText())
-        .put("jdbc_url", String.format("jdbc:oracle:thin:@//%s:%s/%s",
+    List<String> additionalParameters = new ArrayList<>();
+
+    JsonNode encryption = config.get("encryption");
```

Comment:
Would it work with older versions without `encryption`? New version of connector should be backward compatible with previous versions.

Reply:
> Would it work with older versions without `encryption`? New version of connector should be backward compatible with previous versions.

removed encryption from reqired fields